cellarium/cas/settings.py

```diff
@@ -7,6 +9,8 @@
 MAX_CHUNK_SIZE_SEARCH_METHOD = 500
 CELLARIUM_CLOUD_BACKEND_URL = "https://cellarium-cloud-api.cellarium.ai"
 
+logging.basicConfig(level=logging.INFO, format="* [%(asctime)s.%(msecs)03d] %(message)s", datefmt="%H:%M:%S")
```

Comment:
Fine for now but you might want to be careful fundamentally changing how logging happens if this gets used in another environment

Reply:
Agreed.

Reply:
Done.